docs/applications/nube-sdk/events.md

```diff
@@ -358,6 +356,62 @@ nube.send("ui:slot:set", () => ({
 }));
 ```
 
+## `location:updated`
```

Comment:
Following today's checkpoint convo:
- Do we feel necessary to explain that there're other occasions where we send this `location:updated` event ? Like when scrolling products and so?
- Or do we wait until other event marks this state update?